### PR TITLE
e2e: move funs of framework/deviceplugin to e2e_node

### PR DESCRIPTION
**Pull request by haosdent**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Move funs of framework/deviceplugin to e2e_node.

**Which issue(s) this PR fixes**:

Ref #86052

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by haosdent** [comment]
cc @SataQiu

**Review by neolit123 — COMMENTED** [review]
/ok-to-test
/kind cleanup
/lgtm
/approve

File: test/e2e_node/device_plugin_test.go [hunk 1]
```diff
@@ -20,17 +20,21 @@ import (
 	"path/filepath"
 	"time"
 
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/kubernetes/test/e2e/framework/testfiles"
```
Comment (oomichi) [inline test/e2e_node/device_plugin_test.go]: pull-kubernetes-bazel-build job is failed like
```
Execution platform: //:rbe_with_network
compilepkg: missing strict dependencies:
	/b/f/w/test/e2e_node/device_plugin_test.go: import of "k8s.io/api/apps/v1"
	/b/f/w/test/e2e_node/device_plugin_test.go: import of "k8s.io/apimachinery/pkg/runtime/serializer"
No dependencies were provided.
Check that imports in Go sources match importpath attributes in deps.
ERROR: /home/prow/go/src/github.com/kubernetes/kubernetes/test/e2e_node/BUILD:106:1 GoCompilePkg test/e2e_node/linux_amd64_stripped/_go_default_test-cgo%/k8s.io/kubernetes/test/e2e_node.a failed (Exit 1): builder failed: error executing command 
```
I guess we need to run `./hack/update-bazel.sh` and apply the change into this PR.

Comment (haosdent) [reply test/e2e_node/device_plugin_test.go]: thx @oomichi fixed as well.

**Comment by haosdent** [comment]
thx @SataQiu !

**Review by SataQiu — APPROVED** [review]

**Review by SataQiu — COMMENTED** [review]
/lgtm
/test pull-kubernetes-e2e-gce-100-performance

**Pull request merged**